src/components/events/createEvent/EventsAdditionalInfo.jsx

```diff
@@ -54,4 +63,9 @@ const EventsAdditionalInfo = () => {
   );
 };
 
+EventsAdditionalInfo.propTypes = {
+  states: PropTypes.objectOf(PropTypes.any).isRequired,
+  setStates: PropTypes.objectOf(PropTypes.any).isRequired,
```

Comment:
🚫 **[eslint]** <[react/forbid-prop-types](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/forbid-prop-types.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Prop type "any" is forbidden